### ISSUE TITLE
Minecraft mod

### ISSUE THREAD

**Issue by MattsMc** [SteelSeries/gamesense-sdk#2]
You may want to put a check if the item the player holding is a tool, at the moment it acts when no item in hand/item with no durability should be able using instanceof ItemTool iirc or you could check if the item has a durability for future compatibility

https://github.com/SteelSeries/gamesense-sdk/blob/master/examples/minecraftforge1.8/src/main/java/com/sse3/gamesense/GameSenseEventReceiver.java#L117

**Comment by davidtibbetts** [comment]
The Hoe and Shears aren't of type ItemTool, which is why I just switched on the name. I suppose I can check if getItem() of the ItemStack from getHeldItem() is valid, then switch on the type instead of getting the name and switching on the string.

**Comment by MattsMc** [comment]
Yes but it seems that even though you switch on the item string it still
seems to say that your empty hand is almost/completely out of durability.
This is where i was going with my issue :p
